kuma/search/signals.py

```diff
@@ -22,7 +22,7 @@ def render_done_handler(**kwargs):
         if outdated:
             log.info('Found a newer index and scheduled '
                      'indexing it after promotion.')
-        doc_pks = set(doc.other_translations.values_list('pk', flat=True))
+        doc_pks = set([item.pk for item in doc.other_translations])
```

Comment:
In the previous version, ``doc.other_translations`` would return a queryset instead of a list. so we can just get the ``pk`` in list. The previous version had the correct result, but it was not sorted and document was in random order.
Moreover, if you take closer look in the ``other_translations`` model function, you will see two sql quires was being made to generate this in the past solution. this solution works with only single query